src/createServer.js

```diff
@@ -1,3 +1,70 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+// Write code here
+// Also, you can create additional files in the src folder
+// and import (require) them here
+
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+const createServer = () => {
+    const server = http.createServer((request, response) => {
+    const normalizedUrl = new URL(request.url, `http://${request.headers.url}`);

```

Comment:
```suggestion
    const server = http.createServer((request, response) => {
      const normalizedUrl = new URL(request.url, `http://${request.headers.url}`);

```
nested code should not have same indentation